week-2/L-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ var everyone = [
   "Swathi"
 ];
 
-var firstFive; // complete this statement
-var lastFive; // complete this statement
+var firstFive = everyone.slice(0,5); // complete this statement
+var lastFive =everyone.slice(2,7); // complete this statement
```

Comment:
Hmm, this assumes the array has 7 elements. Can we make it work even if the array has 14, 20, or any number of elements?